pytest.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
-addopts = --ds=config.settings.test --reuse-db
+# Test Database re-usable settings.
+# https://pytest-django.readthedocs.io/en/latest/database.html#reuse-db-reuse-the-testing-database-between-test-runs
+
+addopts = --ds=config.settings.test --create-db
```

Comment:
이거 create-db가 더 좋아?
공식문서에는 이렇게 나와있던데! "When you alter your database schema" 부분
A good way to use --reuse-db and --create-db can be:

Put --reuse-db in your default options (in your project’s pytest.ini file):

[pytest]
addopts = --reuse-db
Just run tests with pytest, on the first run the test database will be created. The next test run it will be reused.

When you alter your database schema, run pytest --create-db, to force re-creation of the test database.

Reply:
그러게 나도 공식문서에서 default를 --reuse-db로 하는걸 추천하는건 봤는데
저 설정대로 하면 board/tests/test_views_topic_api/test_retrieve_post_ordering_by_query 부분에서 자꾸 오류가 발생해.

Reply:
![image](https://user-images.githubusercontent.com/89024993/184521474-10b01edd-1d72-4775-bd78-1af002f516a4.png)

Reply:
test 실행되기 전에 Post DB를 초기화 해주는 코드 한줄 추가했더니 문제는 없네.
pytest 공식 문서대로 --reuse-db로 가보자 그럼~